.github/workflows/main.yml

```diff
@@ -24,6 +24,9 @@ jobs:
       run: |
         npm install
         npm run build
+        cp src/tests/playwright/pages/live-suite/style.css _site/tests/playwright/pages/live-suite/style.css
+        mkdir _site/tests/playwright/pages/reference
+        cp src/tests/playwright/pages/reference/440@48k-sine-octx.json _site/tests/playwright/pages/reference/440@48k-sine-octx.json
```

Comment:
Similar, could we add  `src/tests/playwright/pages/reference/*.json` to [.eleventy.js](https://github.com/GoogleChromeLabs/web-audio-samples/blob/main/.eleventy.js#L46) instead?